app/models/PropertyLink.scala

```diff
@@ -64,6 +64,9 @@ case class ApiAssessment(
 ) {
   def isWelsh: Boolean = billingAuthorityCode.exists(_.startsWith("6"))
   def isDraft: Boolean = listType == ListType.DRAFT
+
+  def isCurrent: Boolean = listType == ListType.CURRENT && currentFromDate.nonEmpty && currentToDate.isEmpty
+  def isPrevious: Boolean = !isDraft && !isCurrent
```

Comment:
I don't see this `isPrevious` used anywhere else, it's a new def so should be referenced somewhere else in this PR but it isn't

Reply:
also `isCurrent` appears to only be used on an assessment, not on a property link... so not sure if either of these two are used

Reply:
isCurrent is used in this file `[valuationTab.scala.html](https://github.com/hmrc/voa-property-linking-frontend/pull/1164/files#diff-c783eeec6d6bd34ccb20386763640110d0f020d6f028bfabeaa22c8dbc105a3b)` I forgot to remove isPrevious (i didn't need it in the end)

Reply:
<img width="322" alt="image" src="https://user-images.githubusercontent.com/1763911/196654812-df8373cd-adb9-41f0-bb26-0dd7c5d55df3.png">
i can only see it mentioned in `assessment.isCurrent` not in propertyLink. is `assessment` actually a property link?